helm/ingress-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: ingress-operator
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
```

Comment:
does it need access to all configmaps or a very specific ones? If it's a limited set we can limit it.

Reply:
well there is a config map to configure the operator but at the same time it updates and gets configmaps/services dynamically (at least it is what I understand reading the code)

Reply:
ok, then let's leave this